spring-kafka/src/main/java/org/springframework/kafka/annotation/RetryTopicConfigurationProvider.java

```diff
@@ -96,19 +100,34 @@ public RetryTopicConfigurationProvider(@Nullable BeanFactory beanFactory, @Nulla
 		this.resolver = resolver;
 		this.expressionContext = expressionContext;
 	}
+
 	@Nullable
 	public RetryTopicConfiguration findRetryConfigurationFor(String[] topics, Method method, Object bean) {
-		RetryableTopic annotation = MergedAnnotations.from(method, SearchStrategy.TYPE_HIERARCHY,
-					RepeatableContainers.none())
-				.get(RetryableTopic.class)
-				.synthesize(MergedAnnotation::isPresent)
-				.orElse(null);
+		return findRetryConfigurationFor(topics, method, null, bean);
+	}
+
+	@Nullable
+	public RetryTopicConfiguration findRetryConfigurationFor(String[] topics, @Nullable Method method,
```

Comment:
Why does it need to be `public`? If it's valid, add the `since` tag and a note in `whats-new` as this is a new public API.

Reply:
> note in whats-new as this is a new public API.

It doesn't seem useful for a user to know how to get `RetryTopicConfiguration`. Unless we write a separate section that describes how to manually register main and retry endpoints using the `RetryTopicConfigurer`.

Reply:
Fix it.